### DISCUSSION TITLE
Web Clipper for Evernote

### DISCUSSION THREAD

**Discussion by Bidroiddroid** [webclipper/web-clipper#727] [General]
Purchase the premium version of the web clipper, but miss saving on EVERNOTE.

Do you know when you will support evernote for storage?

Thanks

**Comment by DiamondYuan** [comment]
I think  web clipper of evernote  is good enough .
Will not consider supporting evernote in a short time .

**Reply by CC-bit** [reply]
just want less extension installed😂

**Comment by DiamondYuan** [comment]
I want to focus on other features.

**Comment by Tesgin** [comment]
FWIW, I would be VERY interested in clipping to Evernote (EN).  The most recent versions of the EN clipper have been dumbed down. Not as good as they used to be

**Comment by bytrangle** [comment]
For a long time, Evernote Web Clipper was the best in business. Nothing even came close. I agree that the most recent versions have been dumbed down. FWIT, I understand the maintainer's decision to not include support for web clipper.
